managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/ManagedLedgerImpl.java

```diff
@@ -3639,6 +3664,7 @@ private void checkManagedLedgerIsOpen() throws ManagedLedgerException {
     }
 
     synchronized void setFenced() {
+        log.info("{} Moving to Fenced state", name);
```

Comment:
This is not a "problem".
it may happen, and we are handling it safely.
there is nothing that the sysadmin should be afraid of.

we should log "WARN" or "ERROR" when there is something bad, and you have to take extra care